### ISSUE TITLE
Script not working

### ISSUE THREAD

**Issue by ashley-stendel** [spring-cloud-stream-app-starters/scriptable-transform#5]
Hello, 

Is there any way to pass a script file? 
We are trying to convert json to xml.... and we ran the following stream definition and no transformation occurs.  (scriptable-transform-processor-rabbit 1.3.1 RELEASE)

```
 http | scriptable --language=python --script="def json2xml(json_obj, line_padding=""""):\n result_list = list()\n\n json_obj_type = type(json_obj)\n\n if json_obj_type is list:\n for sub_elem in json_obj:\n result_list.append(json2xml(sub_elem, line_padding))\n\n return "" "".join(result_list)\n\n if json_obj_type is dict:\n for tag_name in json_obj:\n sub_obj = json_obj[tag_name]\n result_list.append(""%s<%s>"" % (line_padding, tag_name))\n result_list.append(json2xml(sub_obj, "" "" + line_padding))\n result_list.append(""%s</%s>"" % (line_padding, tag_name))\n\n return "" "".join(result_list)\n\n return ""%s%s"" % (line_padding, json_obj)\nreturn json2xml(payload)" | log
```

Any insights?
Thanks so much!

**Comment by artembilan** [comment]
I think this is really a valid request and we should support something like `scriptLocation` along side with the `script` as a mutually exclusive option.

Meanwhile consider to use https://github.com/spring-cloud-stream-app-starters/python/tree/master/spring-cloud-starter-stream-processor-python-jython instead.

**Comment by ashley-stendel** [comment]
Thanks! We'll give that a try.

**Comment by ashley-stendel** [comment]
Hi!

We are trying to use python-jython 1.0.1.RELEASE. The file is executable. 
we created the following stream 
`stream create --name converter --definition " http | python-jython --git.uri=https://github.com/ritceynoah/json2xml --jython.script=json2xml.py | log"`

and 

`stream create --name converter --definition " http | python-jython --jython.script=~/Documents/json2xml.py | log"`

We are deploying to kubernetes and we get the following error in our logs.
`Caused by: java.lang.IllegalArgumentException: Cannot access script json2xml.py`

I think it can't find the file.... any suggestions on where we went wrong?

Thank you!
